exercises/practice/zebra-puzzle/tests/Tests.elm

```diff
@@ -0,0 +1,20 @@
+module Tests exposing (tests)
+
+import Expect
+import Test exposing (Test, describe, skip, test)
+import ZebraPuzzle
+
+
+tests : Test
+tests =
+    describe "ZebraPuzzle"
+        [ test "resident who drinks water" <|
```

Comment:
This code / tests are a bit strange, as the answer is already known, so it would be very easy to pass the tests just by returning these values, which is what you would do if you were using tdd. This is a standard exercise though, right? So probably not something we should worry about here.

Reply:
Right, this is a standard exercise, and incredibly easy to pass if you cheat.
I was thinking of adding an analyzer check to make sure that the functions are not exactly `drinksWater = Just Norwegian`, but beyond that, I'm not too worried about it.